### PR TITLE
UISlider fix

### PR DESCRIPTION
**Pull request by MikhailMCraft**
Fixed `UISlider` and started the options menu.

Note there is still an issue with `UIElement` click detection where the `UISlider` will snap to the last clicked position if an overlayed `UIElement` was clicked just prior to the `UISlider` - look into this or I will fix later.

### CODE REVIEW

File: Internals/Common/Input.cs [hunk 1]
```diff
@@ -20,6 +20,9 @@ public static class Input
         public static GamePadState OldGamePadSnapshot { get; internal set; }
 
         public static void HandleInput(PlayerIndex pIndex = PlayerIndex.One) {
+            OldKeySnapshot = CurrentKeySnapshot;
+            OldMouseSnapshot = CurrentMouseSnapshot;
+            OldGamePadSnapshot = CurrentGamePadSnapshot;
```
Comment (RighteousRyan1) [inline Internals/Common/Input.cs:25]: have you tested that moving these here works?

File: TankGame.cs [hunk 2]
```diff
@@ -41,8 +41,8 @@ public class GameConfig
         public static bool BorderlessWindow { get; set; } = true;
```
Comment (RighteousRyan1) [inline TankGame.cs:41]: why are the graphics settings still static even though the other settings are not?

File: GameContent/WPTR.cs [hunk 3]
```diff
@@ -159,6 +157,7 @@ internal static void Update()
 
         public static int tankToSpawnType;
         public static int tankToSpawnTeam;
+        private static int _delay;
```
Comment (RighteousRyan1) [inline GameContent/WPTR.cs:160]: remove this? i dont see a use for it

**Review by RighteousRyan1 — CHANGES_REQUESTED** [review]
Overall fine, just make these changes.

File: Internals/Common/Input.cs (hunk 1 above)
Comment (MikhailMCraft) [reply Internals/Common/Input.cs:25]: It works, prior to this my checks in `WPTR.DoRender` for `UIElement` mouse events would not work when using the newly added `Input.OldMouseX` but does now.

**Pull request merged**